### PR TITLE
Setup a linting tool and style guide

### PR DESCRIPTION
**Pull request by nadralia**
**What does this PR do?**
 

- This PR is for setup linting tool and style guide. 

**Description of Task to be completed?**
- I installed autopep8, to automatically formats Python code to conform to the PEP 8 style guide.
- I installed flake8, One of the useful feature I found with flake8 you can run it for specific type of 
   warnings, errors, etc. just like in the file .flake8 am using it to limit the maximum line length to 80

**How should this be manually tested?**
 

- Just install the dependencies using `pip install -r requirements.txt` and you can also install pylint using vs code extensions packages. Just make sure you re on your virtual environment.

##Reviewers

- @fodongkara @engjames @JamesMudidi @3Nakajugo @llwasampijja @sanya-kenneth

### CODE REVIEW

**Comment by fodongkara** [comment]
@nadralia can you explain to us why you used each of the tools you added? Some seem to be doing duplicate work.

**Comment by nadralia** [comment]
> @nadralia can you explain to us why you used each of the tools you added? Some seem to be doing duplicate work.

- I installed autopep8, to automatically formats Python code to conform to the PEP 8 style guide.
- I installed flake8, One of the useful feature I found with flake8 you can run it for specific type of 
   warnings, errors, etc. just like in the file .flake8 am using it to limit the maximum line length to 80

**Comment by nadralia** [comment]
> > @nadralia can you explain to us why you used each of the tools you added? Some seem to be doing duplicate work.
> 
> * I installed autopep8, to automatically formats Python code to conform to the PEP 8 style guide.
> * I installed flake8, One of the useful feature I found with flake8 you can run it for specific type of
>   warnings, errors, etc. just like in the file .flake8 am using it to limit the maximum line length to 80

**Comment by nadralia** [comment]
> > > @nadralia can you explain to us why you used each of the tools you added? Some seem to be doing duplicate work.
> > > yeah some are doing duplicate work let me remove.
> > 
> > 
> > 
> > * I installed autopep8, to automatically formats Python code to conform to the PEP 8 style guide.
> > * I installed flake8, One of the useful feature I found with flake8 you can run it for specific type of
> >   warnings, errors, etc. just like in the file .flake8 am using it to limit the maximum line length to 80

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[flake8]
+ignore = W391
+max-line-length = 80
+exclude = */tests/*
```
Comment (skapeyi) [inline .flake8]: Why are we ignoring tests?

Comment (nadralia) [reply .flake8]: > Why are we ignoring tests?

@skapeyi  The exclude parameter is used to ignore file/dirs. In this case i made a mistake. But for this project i removed .flake8 , some we re not using it and hence we are not excluding tests.

Comment (fodongkara) [reply .flake8]: @nadralia remove the line `exclude = */tests/*` so that I can merge this.

Comment (fodongkara) [reply .flake8]: @nadralia never mind. I see you removed .flake8

**Pull request merged**